src/main/java/mate/controller/CreateCarController.java

```diff
@@ -0,0 +1,43 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/create/car")
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice): Let’s use one style for creating URLs. Try putting at the first position in URL’s name the name of the model with which we work.
If we work with the manufacturer - let’s put it here: /[manufacturers]/…/… It is good practice to use plurals.